modules/ROOT/partials/configuration/dialog_persistent.adoc

```diff
@@ -0,0 +1,104 @@
+[[persistent]]
+=== `persistent`
+
+include::partial$misc/admon-requires-6.6v.adoc[]
+
+The `+persistent+` property, when set to `+true+`, allows an inline dialog to stay open when the dialog no longer has focus.
+
+If an end-user gives another part of the {productname} editor focus (for example, the document editor, or a menu), the inline dialog will stay open.
+
+Setting the property to `+true+` does not over-ride normal mechanisms for closing a dialog, such as clicking the _Close_ button or pressing the *Esc* key.
+
+The property has a default value of `+false+`.
+
+NOTE: If an inline dialog is open and focus is switched away from the {productname} editor entirely (by, for example, the end-user switching focus to an available host-browser UI element), an open inline dialog will remain open regardless of the `+persistent+` property’s value. This was the behavior prior to this property being available and continues to be the behavior.
+
+*Type:* `+String+`
+
+*Possible values:* `+true+`, `+false+`
+
+*Default value:* `false`
+
+=== Example: using `persistent`
+
+[source,js]
+----
+tinymce.activeEditor.windowManager.open({
+  title: 'Dialog Title', // The dialog's title - displayed in the dialog header
+  body: {
+    type: 'panel', // The root body type - a Panel or TabPanel
+    items: [ // A list of panel components
+      {
+        type: 'htmlpanel', // an HTML panel component
+        html: 'Panel content goes here.'
+      }
+    ]
+  },
+  buttons: [ // A list of footer buttons
+    {
+      type: 'submit',
+      text: 'OK'
+    }
+  ]
+  { inline, persistent: 'true' }
+});
+----
+
+=== Example: using `focus` to refocus an open but de-focused dialog
+
+Because a dialog with an `{ inline, persistent: 'true' }` property does not close when the dialog loses focus, it is possible for an end-user to open a further instance of the dialog.
+
+To prevent this, use the xref:apis/tinymce.editor/#focus[`focus`] method in the xref:apis/tinymce.windowmanager/#return-value[`DialogInstanceAPI`]. This transfers focus to the open dialog rather than creating a new instance of the same dialog.
```

Comment:
The integrator would have to implement a check to see if there is already an instance of their dialog open, and only focus when there is, rather than open a second dialog.

Reply:
> This is demonstrated in the below example, but should also be noted here.

And now [it is](https://github.com/tinymce/tinymce-docs/commit/2ace6407ff3cfe9feb2c2863524d5f85260d69ea)